app/transform/k8s/node_transformer.py

```diff
@@ -35,9 +35,10 @@ def add_cpu_resource(self, node_id: IRI, timestamp: int) -> None:
             cpu_capacity_id = cpu_id.dot("Capacity")
             self.add_measurement(
                 cpu_id.dot("Capacity"),
-                "Capacity CPU",
+                "CPU.Capacity",
```

Comment:
There are a lot of places where you change this metric's name (and similar metrics). Maybe it is better to create constants. This will allow us to make this kind of change faster next time.

Reply:
fixed